joomla/joomla-cms
# [4.0] Adding custom-select class to layout/choose layout fields

Following up on https://github.com/joomla/joomla-cms/pull/21850

### Summary of Changes
Added `custom-select` class


### Testing Instructions
See https://github.com/joomla/joomla-cms/pull/21850#issuecomment-416050895

Looks before and after patch when edit modules, + some for contact, tags, newsfeeds

### Documentation required
Yes
